doc/developers_guide/developers_guide.md

```diff
@@ -0,0 +1,11 @@
+# Developers Guide
+
+## Building the native image
+
+Current try to build a native image:
+
+```shell
+ ~/programme/graalvm-ce-java11-21.1.0/bin/native-image --static -cp target/parquet-edml-generator.jar com.exasol.edmlgenerator.parquet.Main --no-fallback
```

Comment:
You could also generalize the native-image path. 

Please also add if there are steps to install the native image.